app/template.rb

```diff
@@ -0,0 +1,27 @@
+# Javascript
+directory 'app/javascript'
+
+insert_into_file 'app/javascript/packs/application.js', after: %r{require\("channels"\)\n} do
+  <<~EOT
+  
+    import 'translations/translations';
+    
+    import 'initializers/';
+    import 'screens/';
+  EOT
+end
+
+# Stylesheets
+remove_file 'app/assets/stylesheets/application.css'
+directory 'app/assets/stylesheets'
+
+# Controllers
+directory 'app/controllers/concerns'
+inject_into_class 'app/controllers/application_controller.rb', 'ApplicationController' do
+  "  include Localization\n"
```

Comment:
This extra space seems akward. How about to use the tab `\t`? or using two-character specs?

Reply:
Updated the test. I think later I would like to follow the [HEREDOC active_support layout](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/IndentHeredoc
) in the rubocop guide for the template text. 

Currently we're using `<<-EOT` for keeping the indent and `<<~EOT` for stripping the indent. But maybe we should use `<<-TEXT` and `<<-TEXT.strip_heredoc`  instead. So we always use dash. 🤔 

If you agree I can create the ticket in `4.0.0` and create a PR for it.

Reply:
I agree. SO let's fix that in another PR.